mpdev/internal/apply/types.go

```diff
@@ -0,0 +1,47 @@
+package apply
+
+
+const API_VERSION = "dev.marketplace.cloud.google.com/v1alpha1"
+var TypeMapper = map[TypeMeta]func() Resource{
```

Comment:
Turn this into a private variable (so that it's not accidentally modified). Instead, export a function that does the mapping.

Reply:
Fixed